scripts/apps/authoring/tests/authoring.spec.ts

```diff
@@ -90,22 +92,6 @@ describe('authoring', () => {
             expect(ITEM._locked).toBe(true);
         }));
 
-    it('unlocks a locked item and locks by current user',
```

Comment:
why removing this one?

Reply:
`expect('a', 'b')` does not check anything - it should be `expect('a').toBe('b')`. When I corrected the syntax it failed since (`$location.path()` would evaluate to an empty string) so I removed it together with all other tests which haven't worked for years.